rosidl_generator_cpp/resource/msg__struct.hpp.em

```diff
@@ -43,6 +43,7 @@ cpp_full_name_with_alloc = '%s<ContainerAllocator>' % (cpp_full_name)
 #include <memory>
 #include <string>
 #include <vector>
+#include <rosidl_generator_cpp/bounded_vector.hpp>
```

Comment:
Does this need to be imported for all messages, or just ones that contain fields with bounded arrays?